sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKeyBuilder.java

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Undefined;
+import com.azure.cosmos.implementation.routing.PartitionKeyInternal;
+import com.azure.cosmos.util.Beta;
+import com.azure.cosmos.util.Beta.SinceVersion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Beta(value = SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public class PartitionKeyBuilder {
+    private final List<Object> partitionKeyValues;
+
+    /**
+     * Constructor. CREATE a new instance of the PartitionKeyBuilder object.
+     */
+    public PartitionKeyBuilder() {
```

Comment:
Please add the @Beta annotation to all the public APIs for this class.

Reply:
Fixed as suggested